### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by StuartIanNaylor** [nekon1987/fpool3-keras-implementation#1]
Hi nekon apols but total noob here.

I things to run and create the model.
How do I test from there?

Many thanks.

**Comment by nekon1987** [comment]
Hi Stuart, most of the process is described in an article I have written about this https://blog.aspiresys.pl/technology/building-jarvis-nlp-hot-word-detection/ If this isnt enough let me know. It makes things easier if you run it from PyCharm

**Comment by StuartIanNaylor** [comment]
https://blog.aspiresys.pl/technology/building-jarvis-nlp-hot-word-detection/

Yeah I followed https://blog.aspiresys.pl/technology/building-jarvis-nlp-hot-word-detection/ but got lost along the way and was thinking maybe just a bit of noob dust wouldn't go amiss :)

I will check PyCharm.

I have actually been running the speech_commands example from tensorflow and just wanted to compare against yours.
I have been playing with models and as well as keyword I added 2 other labels of one-sylable & two-sylable as the 'visual' keyword from the google command set is three sylable.
It seemed to make the keyword score better and also you could use the other scores as reference adjusment.
Does that make sense as can you affect the model with choice of other classification?

**Comment by nekon1987** [comment]
Any questions you have I will try to answer. If I understand this one correctly - you are wondering if sylab length is affecting the classification and the answer is yes, the longer the voice sample the better analysis can the network do, Thats why most companies create hot words like Hey Siri or Hey Google, Alexa is different because its already long and distinguishable

**Comment by StuartIanNaylor** [comment]
No I picked the word visual because of that but also had 2 further labels of onesylable and twosylable with respective words and it seemed to make the word visual easier to detect than just my first run of just visual and unknown.
Thinking about it the extra labels would of just added more data to the model with the default settings each time and prob why I guess. 
Thanks for the reply though.
